halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/registry/v1/GitProfileReader.java

```diff
@@ -72,7 +81,27 @@ public Versions readVersions() throws IOException {
 
   @Override
   public InputStream readArchiveProfile(String artifactName, String version, String profileName) throws IOException {
-    throw new UnsupportedOperationException();
+    Path profilePath = Paths.get(profilePath(artifactName, version, profileName));
```

Comment:
Does this resolve to the local packer contents in the git repo?

Reply:
Yes, exactly.

Reply:
Perfect